pkg/sdn/plugin/subnets.go

```diff
@@ -220,7 +221,15 @@ func (master *OsdnMaster) watchSubnets() {
 					log.Errorf("Error in deleting annotated subnet from master, name: %s, ip %s: %v", name, hostIP, err)
 					return nil
 				}
-				err = master.addNode(name, hostIP)
+				var hsAnnotations map[string]string
+				if vnid, ok := hs.Annotations[osapi.FixedVnidHost]; ok {
```

Comment:
This check is only valid for multi-tenant sdn plugin, should ignore for subnet plugin (osapi.IsOpenShiftMultitenantNetworkPlugin)

If we know that F5 with multi-tenant sdn plugin needs this hack, may be enforce osapi.FixedVnidHost to be present when osapi.AssignHostSubnetAnnotation is passed?